include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -49,164 +49,25 @@ namespace parallel
   namespace fullydistributed
   {
     /**
-     * Configuration flags for fully distributed Triangulations.
-     * Settings can be combined using bitwise OR.
-     *
-     * @author Peter Munch, 2019
+     *  Using directive for backwards-compatibility.
+     *  @deprecated Use construction::Settings instead.
      */
-    enum Settings
-    {
-      /**
-       * Default settings, other options are disabled.
-       */
-      default_setting = 0x0,
-      /**
-       * This flags needs to be set to use the geometric multigrid
-       * functionality. This option requires additional computation and
-       * communication.
-       */
-      construct_multigrid_hierarchy = 0x1
-    };
+    using Settings = construction::Settings;
```

Comment:
I'm not sure we want to keep this in. We had no releases yet with p::f::t, so there is no need to keep backward compatibility. But that's just my two cents. @kronbichler ?

Reply:
I agree, we don't need backwards compatibility at this point.